install/install.sh

```diff
@@ -60,27 +55,53 @@ curl -o ${archiveFile} ${baseurl}/${archiveFile}
 mkdir ${tempdir}
 tar -xf ${archiveFile} -C ${tempdir}
 
-installLocation="/usr/local/bin"
-cp ${tempdir}/docker-credential-acr-${os} $installLocation
-chmod +x $installLocation/docker-credential-acr-${os}
+defaultInstallLocation="/usr/local/bin"
+## This is an attempt to check whether we are running the script interactively
+## It can't detect when user pipes the script into bash, in that case default would still be used
+case $- in
```

Comment:
Right now if user pipe the script to /bin/bash, the script would just assume the user did not input anything but was unable to know the user cannot interactively input any value. If anyone know how to fix this, let me know